src/main.rs

```diff
@@ -111,37 +112,37 @@ use tls::TlsOption;
 use traits::Controller;
 
 docopt!(Args derive Debug, "
-Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-d <profile_path>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--remote-name <hostname>] [--dns-domain <domain>] [-c <namespace;key;value>]...
+Usage: foxbox [-v] [-h] [-l <hostname>] [-p <port>] [-w <wsport>] [-d <profile_path>] [-r <url>] [-i <iface>] [-t <tunnel>] [-s <secret>] [--disable-tls] [--dns-domain <domain>] [--dns-api <url>] [--tunnel-frontend <frontend>] [-c <namespace;key;value>]...
 
 Options:
     -v, --verbose            Toggle verbose output.
-    -l, --local-name <hostname>    Set local hostname. Linux only. Requires to be a member of the netdev group.
+    -l, --local-name <hostname>    Set local hostname. [default: foxbox]
     -p, --port <port>        Set port to listen on for http connections. [default: 3000]
     -w, --wsport <wsport>    Set port to listen on for websocket. [default: 4000]
     -d, --profile <path>     Set profile path to store user data.
     -r, --register <url>     Change the url of the registration endpoint. [default: http://localhost:4242]
```

Comment:
Since you are using `knilxof.org` as default for other options, let's take the chance and make this one also default to `http://knilxof.org:4242`. This way people not reading the docs and simply running `cargo run` will still be able to run it successfully so it can work with the demo app.